src/main/java/com/google/cloud/pubsublite/spark/PslReadDataSourceOptions.java

```diff
@@ -63,6 +63,8 @@
 public abstract class PslReadDataSourceOptions implements Serializable {
   private static final long serialVersionUID = 2680059304693561607L;
 
+  private SubscriberServiceClient subscriberServiceClient = null;
```

Comment:
I think this is now non-serializable- you need to mark this as transient

Reply:
Done